Pulley/DrawerContentViewController.swift

```diff
@@ -93,7 +93,7 @@ extension DrawerContentViewController: PulleyDrawerViewControllerDelegate {
     }
     
     func supportedDrawerPositions() -> [PulleyPosition] {
-        return PulleyPosition.all // You can specify the drawer positions you support. This is the same as: [.open, .partiallyRevealed, .collapsed, .closed]
+        return PulleyPosition.artsy // You can specify the drawer positions you support. This is the same as: [.open, .partiallyRevealed, .collapsed, .closed]
```

Comment:
we actually don't need to maintain an artsy specific array of drawer positions as we're using all of the standard ones now.